CHANGELOG.rst

```diff
@@ -0,0 +1,4 @@
+## Changes since 2021-08-17T07:33:29.505Z
+
+ * @stsewd: Make it easy to test locally (#83)
+
```

Comment:
We usually run the automatic changelog command when doing a release

Reply:
Aha, yeah my bad. I was following the docs on "_[Local testing](https://another-test-stsewd.readthedocs.io/projects/readthedocs-sphinx-search/en/latest/development.html#local-testing)_" and continued following onto "_Releasing_" without realising 😬

Reply:
I rebased git history and removed this commit.